### PR TITLE
Add options flow docs to PurpleAir

### PR DESCRIPTION
**Pull request by bachya**
## Proposed change

SSIA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/83440
- Link to parent pull request in the Brands repository: N/A
- This PR fixes or closes issue: N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/purpleair.markdown [hunk 1]
```diff
@@ -17,3 +17,9 @@ ha_config_flow: true
 [PurpleAir](https://www2.purpleair.com/) makes sensors that measure hyper-local air quality data and share it with the public.
 
 {% include integrations/config_flow.md %}
+
+{% include integrations/option_flow.md %}
```
Comment (frenck) [inline source/_integrations/purpleair.markdown:21]: The options flow does not provide a header.

In general, when referring to the options flow, including a list of options using a basic configuration block is helpful.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/purpleair.markdown (hunk 1 above)
Comment (bachya) [reply source/_integrations/purpleair.markdown:21]: > The options flow does not provide a header.

Isn't that what this is? https://github.com/home-assistant/home-assistant.io/blob/7d82e68de0655f3cb794e6727c1b388b22c46799/source/_includes/integrations/option_flow.md?plain=1#L3

Comment (frenck) [reply source/_integrations/purpleair.markdown:21]: Oh darn! You are right!

Sorry about that 👍

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
and the parent has been merged already, awesome 👍

**Pull request merged**